client/js/reducers/progress.js

```diff
@@ -57,10 +57,17 @@ export default (state = initialState, action) => {
       break;
 
     case AssessmentConstants.ASSESSMENT_CHECK_ANSWER_DONE:
-      var checkedResponse = Immutable.fromJS(action.payload);
+
+      var checkedResponses = Immutable.Map();
+
+      action.choiceIds.forEach((id) => {
+        var feedback = Immutable.Map(action.payload);
```

Comment:
Add a comment about why feedback is always action.payload. (We don't yet know what the server response will be so we need to get that data and then revise this line so that the feedback is specific to each answer).